test/hotspot/jtreg/runtime/cds/appcds/SharedRegionAlignmentTest.java

```diff
@@ -31,16 +31,21 @@
  *          disable it if ZGC is used.
  * @bug 8236847
  * @library /test/lib /test/hotspot/jtreg/runtime/cds/appcds/test-classes
+ * @build sun.hotspot.WhiteBox
  * @build Hello
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller sun.hotspot.WhiteBox
  * @run driver jdk.test.lib.helpers.ClassFileInstaller -jar hello.jar Hello
- * @run driver SharedRegionAlignmentTest
+ * @run main/othervm/timeout=240 -XX:+UnlockDiagnosticVMOptions -XX:+WhiteBoxAPI -Xbootclasspath/a:. SharedRegionAlignmentTest
```

Comment:
Is the increase in timeout necessary?

Reply:
I will do a test in debug to check if it should be removed. Thanks.